### PR TITLE
RAD-109: fixed readme plugin import typo

### PR DESCRIPTION
**Pull request by adamkimpton**
- Changed readme file because of incorrect syntax for importing a plugin
- Added .idea/ to our gitignore file so that IntelliJ specific files weren't included in the repository

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,5 +1,6 @@
 # Ignore Gradle project-specific cache directory
 .gradle
+.idea/
```
Comment (thomas-anderson-bsl) [inline .gitignore:3]: Directories can be ignored without the '/'

**Review by thomas-anderson-bsl — CHANGES_REQUESTED** [review]
Requires a fix

File: .gitignore (hunk 1 above)
Comment (adamkimpton) [reply .gitignore]: Fixed this issue!

**Review by thomas-anderson-bsl — APPROVED** [review]
LGTM

**Pull request merged**